tests/performance/results.js

```diff
@@ -8,8 +8,12 @@ const { join } = require( 'node:path' );
 const { median, getResultsFilename } = require( './utils' );
 
 const testSuites = [
+	'admin',
+	'admin-l10n',
```

Comment:
Looks like some tabs vs. spaces discrepancy here

Reply:
Well spotted! Too bad we don't have Prettier or ESLint 🙃  Fixing now...